config/swagger.yaml

```diff
@@ -92,9 +92,7 @@ paths:
             type: object
             properties:
               hash:
-                description: 'Key block hash'
-                # Encoded key block hash 'kh_...'
-                type: string
+                $ref: '#/definitions/EncodedHash'
```

Comment:
The schema object is defined [here](https://github.com/OAI/OpenAPI-Specification/blob/8a387ac71f7f31aaff5b910f1e0e99ad12ff3ef0/versions/2.0.md#schemaObject).

It defines `properties` as object where each value is a schema object.

In particular, this follows [`address` in this example](https://github.com/OAI/OpenAPI-Specification/blob/8a387ac71f7f31aaff5b910f1e0e99ad12ff3ef0/versions/2.0.md#simple-model).

Ok.